.github/workflows/pull_request.yml

```diff
@@ -148,14 +159,17 @@ jobs:
           enable-cache: true
 
       - name: Generate requirements file
-        run: python scripts/generate_dependencies.py maximum
+        run: python scripts/generate_dependencies.py ${{ matrix.dependency-set }}
 
       - name: Install dependencies
         run: |
           uv pip install --system --no-deps .
           uv pip install --system -r requirements.txt
           uv pip install --system pytest psutil
-          # (No onnx — Python 3.13)
+          # onnx isn’t available yet on 3.13
+          if [[ "${{ matrix.python-version }}" != "3.13" ]]; then
+            uv pip install --system onnx
+          fi
```

Comment:
onnx 1.18 is out and works on 3.13, but can also do this in a follow-up PR for consistency (also need to remove xfail in the tests).